__tests__/app.test.js

```diff
@@ -308,7 +307,27 @@ describe("app", () => {
             })
         })
 
+    })
+
+    describe("GET /api/articles/:article_id (comment count)", () => {
+        test("should return single object including property of comment_count", () => {
+            return request(app).get("/api/articles/6").expect(200).then(({ body }) => {
+                const { article } = body;
+
+                expect(article).toMatchObject({
+                    title: expect.any(String),
+                    topic: expect.any(String),
+                    author: expect.any(String),
+                    article_id: expect.any(Number),
+                    created_at: expect.any(String),
+                    votes: expect.any(Number),
+                    article_img_url: expect.any(String),
+                    body: expect.any(String),
+                    comment_count: expect.any(Number)
```

Comment:
We have already tested for most of these properties. Only really needed the check for comment_count here